changelog.mdx

```diff
@@ -7,6 +7,39 @@ description: Updates to Chalk!
 
 ---
 
+## October 14, 2024
+
+### SQL explorer in dashboard for datasets
+
+You can now run SQL queries in the dashboard to explore data in [datasets](https://docs.chalk.ai/docs/datasets).
```

Comment:
maybe 'You can now run SQL queries on [dataset] outputs on the Chalk dashboard to explore your data.